src/v4l2_camera.cpp

```diff
@@ -114,6 +126,7 @@ V4L2Camera::V4L2Camera(rclcpp::NodeOptions const & options)
 
         ci->header.stamp = stamp;
         ci->header.frame_id = camera_frame_id_;
+        publish_next_frame_ = publish_rate_ <= 0;
```

Comment:
```suggestion
        publish_next_frame_ = publish_rate_ < 0;
```
condition consistency with l.52

Reply:
Consider we have publish_rate_  = 0.0
In that case, we should publish every frame, otherwise, rclcpp::Rate(0.0) would be very strange.
So here, if publish_rate_ = 0.0, we should still set publish_next_frame_ = true.
So here, instead of be consistent with l.52, I think it should be consistent with l.57.

Reply:
> condition consistency with l.52

Consider we have publish_rate_  = 0.0
In that case, we should publish every frame, otherwise, rclcpp::Rate(0.0) would be very strange.
So here, if publish_rate_ = 0.0, we should still set publish_next_frame_ = true.
So here, instead of be consistent with l.52, I think it should be consistent with l.57.

Reply:
@Mingyu1991 
Thank you for the detailed explanation. I've understood your intention. My concern is the "equal" operation between two floating point values may return a different result from what we expected because of numerical error (e.g., `0.0 == 0.0` may be evaluated as false).  I guess we have three possible options to avoid this unexpected behavior:
- using `int` instead of `double` as the data type for `publish_rate_`
  - I think your code completely works if the data type is integer
- using "almost equal" comparison, such as utilizing `std::numeric_limits<double>::epsilon()`
- treating 0.0 as a special case and converting publish_rate_ into `-1.0` after getting the value (l.51)

This is a very minor case, my apologies for bothering you. But, to avoid potential unexpected behavior, I would appreciate your consideration. Or if you have a different opinion, please let me know.

Reply:
@manato 
Thank you for the advice.
I updated the code following the 2nd suggestion.
Please kindly check it.

Reply:
@Mingyu1991 
Thank you for your prompt handling! My all concerns are gone.